client/src/components/RelatedObjectNotes.js

```diff
@@ -276,11 +276,24 @@ const RelatedObjectNotes = ({
                           .filter(
                             field => !EXCLUDED_ASSESSMENT_FIELDS.includes(field)
                           )
-                          .map(field => (
-                            <p key={field}>
-                              <i>{field}</i>: <b>{jsonFields[field]}</b>
-                            </p>
-                          ))}
+                          .map(field => {
+                            const isRichTextField = utils.isValidHtmlWithTags(
```

Comment:
In principle this should only be true if the dictionary defines the field as having a `richTextEditor` widget.